api/app/signals/apps/services/domain/dsl.py

```diff
@@ -82,10 +82,17 @@ class SignalDslService(DslService):
     context_func = SignalContext()
     signal_manager = SignalManager()
 
-    def process_routing_rules(self, signal):
+    def process_routing_rules(self, signal):  # noqa C901
         ctx = self.context_func(signal)
         rules = RoutingExpression.objects.select_related('_expression', '_department')
         for rule in rules.filter(is_active=True, _expression___type__name='routing').order_by('order'):
+            if (rule._department and rule._user
+                    and not rule._user.profile.departments.filter(id=rule._department.id).exists()):
+                # The user is no longer a member of the department so let's de activate the rule
+                rule.is_active = False
+                rule.save(update_fields=['is_active'])
+                continue  # Something
```

Comment:
Update or remove the `# Something` comment - it is probably left over from working on this.

Reply:
You are right, left over comment. Missed that one.